packages/manager/modules/pci/src/projects/project/failover-ips/order/agora/order.controller.js

```diff
@@ -0,0 +1,59 @@
+import JSURL from 'jsurl';
+import filter from 'lodash/filter';
+import find from 'lodash/find';
+import get from 'lodash/get';
+import head from 'lodash/head';
+import isNumber from 'lodash/isNumber';
+import min from 'lodash/min';
+import { ORDER_URL } from './order.constants';
+// const MESSAGES_CONTAINER_NAME = 'pci.projects.project.failover-ips';
+
+export default class FailoverIpController {
+  /* @ngInject */
+  constructor(
+    $window,
+    OvhApiOrderCloudProjectIp,
+  ) {
+    this.$window = $window;
+    this.OvhApiOrderCloudProjectIp = OvhApiOrderCloudProjectIp;
+  }
+
+  $onInit() {
+    this.ip = {
+      quantity: 1,
+      instance: null,
+      region: null,
+      product: null,
+    };
+
+    this.REGIONS = get(find(get(head(this.products), 'details.product.configurations'), { name: 'country' }), 'values');
```

Comment:
Creating some `const` assignments will improve code readability in this case I think.

It also prevent to use some nested Lodash method calls (four level in this example).

Is that make sense to you?

Example:
```js
const [firstProduct] = this.products;
const configurations = get(firstProduct, 'details.product.configurations')

this.REGIONS = get(find(configurations, { name: 'country' }), 'values');
```